.github/workflows/commisery.yml

```diff
@@ -0,0 +1,32 @@
+# Copyright (C) 2020-2020, TomTom (http://tomtom.com).
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Commisery
+on: 
+  pull_request:
+    types: [edited, opened, synchronize, reopened]
+
+jobs:
+  commit-message:
+    name: Conventional Commit Message Checker (Commisery)
+    runs-on: ubuntu-latest
+    steps:       
+    - name: Check-out the repo under $GITHUB_WORKSPACE
+      uses: actions/checkout@v2
+
+    - name: Run Commisery
+      uses: enarx/commisery-action@master
+      with:
+        token: ${{ secrets.GITHUB_TOKEN }}
+        pull_request: ${{ github.event.number }}
```

Comment:
Why does this action require our `GITHUB_TOKEN`? What permissions are we granting to it?

Reply:
That is unnecessary for a public repo. You should submit a PR to make the token optional.

Reply:
This is unfortunately unfeasible, as this requires putting in a PR for the GitHub Python API library to not require a `GITHUB_TOKEN`. What I think is a better alternative is to simply restrict the access which `GITHUB_TOKEN` provides via: https://github.blog/changelog/2021-04-20-github-actions-control-permissions-for-github_token/